server/public/model/license.go

```diff
@@ -23,8 +23,25 @@ const (
 	LicenseShortSkuE20          = "E20"
 	LicenseShortSkuProfessional = "professional"
 	LicenseShortSkuEnterprise   = "enterprise"
+	LicenseShortSkuPremium      = "premium"
+
+	ProfessionalTier = 1
+	E10Tier          = 1
+	EnterpriseTier   = 2
+	E20Tier          = 2
+	PremiumTier      = 3
```

Comment:
It might be helpful to multiply these values by 10. That would make it easier down the road to add a SKU between the existing ones - for example, "Pro +" in between Professional and Enterprise

Reply:
Done.